{
  "schema_version": "1.4.0",
  "id": "GHSA-4f92-w438-f484",
  "modified": "2024-05-02T12:30:40Z",
  "published": "2024-05-02T12:30:40Z",
  "aliases": [
    "CVE-2024-3955"
  ],
  "details": "URL GET parameter \"logtime\" utilized within the \"downloadlog\" function from \"cbpi/http_endpoints/http_system.py\" is subsequently passed to the \"os.system\" function in \"cbpi/controller/system_controller.py\" without prior validation allowing to execute arbitrary code.This issue affects CraftBeerPi 4: from 4.0.0.58 (commit 563fae9) before 4.4.1.a1 (commit 57572c7).\n\n",
  "severity": [

  ],
  "affected": [

  ],
  "references": [
    {
      "type": "ADVISORY",
      "url": "https://nvd.nist.gov/vuln/detail/CVE-2024-3955"
    },
    {
      "type": "WEB",
      "url": "https://github.com/PiBrewing/craftbeerpi4/issues/132"
    },
    {
      "type": "WEB",
      "url": "https://cert.pl/en/posts/2024/05/CVE-2024-3955"
    },
    {
      "type": "WEB",
      "url": "https://cert.pl/posts/2024/05/CVE-2024-3955"
    }
  ],
  "database_specific": {
    "cwe_ids": [
      "CWE-94"
    ],
    "severity": null,
    "github_reviewed": false,
    "github_reviewed_at": null,
    "nvd_published_at": "2024-05-02T10:15:08Z"
  }
}